src/status_im2/contexts/wallet/send/events.cljs

```diff
@@ -0,0 +1,85 @@
+(ns status-im2.contexts.wallet.send.events
+  (:require
+    [status-im2.constants :as constants]
+    [taoensso.timbre :as log]
+    [utils.money :as money]
+    [utils.number]
+    [utils.re-frame :as rf]))
+
+(rf/reg-event-fx :wallet/suggested-routes-success
+ (fn [{:keys [db]} [suggested-routes timestamp]]
+   (when (= (get-in db [:wallet :ui :send :suggested-routes-call-timestamp]) timestamp)
+     {:db (-> db
+              (assoc-in [:wallet :ui :send :suggested-routes] suggested-routes)
+              (assoc-in [:wallet :ui :send :route] (first (:Best suggested-routes)))
+              (assoc-in [:wallet :ui :send :loading-suggested-routes?] false))})))
+
+(rf/reg-event-fx :wallet/suggested-routes-error
+ (fn [{:keys [db]} [_error]]
+   {:db (-> db
+            (update-in [:wallet :ui :send] dissoc :suggested-routes)
+            (update-in [:wallet :ui :send] dissoc :route)
+            (assoc-in [:wallet :ui :send :loading-suggested-routes?] false))}))
+
+(rf/reg-event-fx :wallet/clean-suggested-routes
+ (fn [{:keys [db]}]
+   {:db (-> db
+            (update-in [:wallet :ui :send] dissoc :suggested-routes)
+            (update-in [:wallet :ui :send] dissoc :route)
+            (update-in [:wallet :ui :send] dissoc :loading-suggested-routes?))}))
+
+(rf/reg-event-fx :wallet/select-send-address
+ (fn [{:keys [db]} [{:keys [address stack-id]}]]
+   (println "dsadadasd" address stack-id)
+   {:db (assoc-in db [:wallet :ui :send :to-address] address)
+    :fx [[:navigate-to-within-stack [:wallet-select-asset stack-id]]]}))
+
+(rf/reg-event-fx :wallet/send-select-token
+ (fn [{:keys [db]} [{:keys [token stack-id]}]]
+   {:db (assoc-in db [:wallet :ui :send :token] token)
+    :fx [[:navigate-to-within-stack [:wallet-send-input-amount stack-id]]]}))
+
+(rf/reg-event-fx :wallet/send-select-amount
+ (fn [{:keys [db]} [{:keys [amount]}]]
+   (js/alert "Not implemented yet")
```

Comment:
Wondering if there is an :fx for `js/alert`, if so - it would be more canonical to use it

Reply:
Not that I am aware of, but would be great to add it!